### PR TITLE
refactor(folderNames): standardise names and add pycache to .gitignore

### PR DESCRIPTION
**Pull request by Matt-Hayter**

### CODE REVIEW

File: frontend/src/components/the-nav-bar/TheNavBar.vue [hunk 1]
```diff
@@ -73,7 +73,7 @@
 </template>
 
 <script>
-import AccountDropdownContents from "@/components/TheNavBar/components/AccountDropdownContents.vue";
+import AccountDropdownContents from "@/components/the-nav-bar/components/AccountDropdownContents.vue";
```
Comment (TJ-io) [inline frontend/src/components/the-nav-bar/TheNavBar.vue:76]: `the-nav-bar` does not follow conventions - call it `nav-bar`

File: frontend/src/views/SEIDR/SEIDR.vue [hunk 2]
```diff
@@ -144,14 +144,14 @@
 </template>
 
 <script>
-import axios from "axios"; //For making client-side http requests
-import TheNavBar from "@/components/TheNavBar/TheNavBar.vue";
-import ConfigBar from "@/components/ConfigBar/ConfigBar.vue";
+import axios from "axios"; //For making http requests
+import TheNavBar from "@/components/the-nav-bar/TheNavBar.vue";
```
Comment (TJ-io) [inline frontend/src/views/SEIDR/SEIDR.vue:148]:
```suggestion
import TheNavBar from "@/components/nav-bar/TheNavBar.vue";
```

File: frontend/src/views/competing-species/CompetingSpecies.vue [hunk 3]
```diff
@@ -109,12 +109,12 @@
 </template>
 
 <script>
-import axios from "axios"; //For making client-side http requests
-import TheNavBar from "@/components/TheNavBar/TheNavBar.vue";
-import ConfigBar from "@/components/ConfigBar/ConfigBar.vue";
+import axios from "axios"; //For making http requests
+import TheNavBar from "@/components/the-nav-bar/TheNavBar.vue";
```
Comment (TJ-io) [inline frontend/src/views/competing-species/CompetingSpecies.vue:113]:
```suggestion
import TheNavBar from "@/components/nav-bar/TheNavBar.vue";
```

File: frontend/src/views/predator-prey/PredatorPrey.vue [hunk 4]
```diff
@@ -108,12 +108,12 @@
 </template>
 
 <script>
-import axios from "axios"; //For making client-side http requests
-import TheNavBar from "@/components/TheNavBar/TheNavBar.vue";
-import ConfigBar from "@/components/ConfigBar/ConfigBar.vue";
+import axios from "axios"; //For making http requests
+import TheNavBar from "@/components/the-nav-bar/TheNavBar.vue";
```
Comment (TJ-io) [inline frontend/src/views/predator-prey/PredatorPrey.vue:112]:
```suggestion
import TheNavBar from "@/components/nav-bar/TheNavBar.vue";
```

File: API/.gitignore [hunk 5]
```diff
@@ -3,3 +3,4 @@
 .elasticbeanstalk/*
 !.elasticbeanstalk/*.cfg.yml
 !.elasticbeanstalk/*.global.yml
+**/__pycache__
```
Comment (TJ-io) [inline API/.gitignore]: Add new line at EOF
```suggestion
**/__pycache__

```

**Review by TJ-io — CHANGES_REQUESTED** [review]
2 small changes

- add new line at EOF
- remove 'the' from nav-bar

File: API/.gitignore (hunk 5 above)
Comment (Matt-Hayter) [reply API/.gitignore]: Done

File: frontend/src/components/the-nav-bar/TheNavBar.vue (hunk 1 above)
Comment (Matt-Hayter) [reply frontend/src/components/the-nav-bar/TheNavBar.vue:76]: There's only one instance of TheNavBar present on each view, and it doesn't accept any props (there's only one variation of the component). In this situation, I believe the [convention](https://v2.vuejs.org/v2/style-guide/?redirect=true#Single-instance-component-names-strongly-recommended) is to use "The"?

Comment (TJ-io) [reply frontend/src/components/the-nav-bar/TheNavBar.vue:76]: Ok thats fair, I have not seen this in Vue 3 docs! Looks good to me

**Review by TJ-io — APPROVED** [review]

**Pull request merged**